woocommerce/google-listings-and-ads
# Free Listings + Paid Ads: Update learn more links, FAQs, CSS disclaimer, submission success modal, and title of free listings settings

### Changes proposed in this Pull Request:

This PR implements the rest parts of **📌 Update other UI changes** in #1610.

- Change the URLs of the learn more links to the formal ones in the ads setup of onboarding flow.
- Update the FAQs in the ads setup.
- Add the disclaimer of Comparison Shopping Service to the accounts setup of onboarding flow.
- Implement the submission success modal on the product feed page after the onboarding is completed along with paid ads.
- Set different titles for the free listings setup of the onboarding and editing pages.

### Screenshots:

#### 📷 Disclaimer of Comparison Shopping Service

![image](https://user-images.githubusercontent.com/17420811/190627390-26dda209-cf82-4e6a-866b-597ad8d345ea.png)

![image](https://user-images.githubusercontent.com/17420811/190627455-113341ff-82ab-4755-a1b9-a77e30b28c5e.png)

#### 📷 FAQs in the ads setup

![image](https://user-images.githubusercontent.com/17420811/190626984-6428f510-233a-4957-9ef0-49040f88e1d3.png)

#### 📷 Submission success modal when the onboarding is completed along with paid ads

![image](https://user-images.githubusercontent.com/17420811/190626820-a0f83cd7-0423-45b3-bc53-8cd2e775d5dc.png)

#### 📷 Different title on the edit free listings page

![image](https://user-images.githubusercontent.com/17420811/190627666-13fad769-db9d-46ee-b4f1-842b291e0225.png)

### Detailed test instructions:

1. Go to the first step of onboarding flow.
2. Check if the copywriting of CSS disclaimer is the same as it is on Figma and if the UI is similar to the visual design.
   - Locally changing [the `isPreconditionReady` here](https://github.com/woocommerce/google-listings-and-ads/blob/3314afc4782763d782d4d212c59d01237a64bce9/js/src/hooks/useGoogleMCAccount.js#L61) to `false` would be easier to check the UI state in the disabled style.
3. Go to the last step of onboarding flow.
4. Click on the two "Learn more" links on the left side to see if the formal pages are opened.
5. View the FAQs to check if the copywriting is the same as it is on Figma.
6. Complete the onboarding flow with sklp ads setup. (This revision doesn't include the implementation of completing with ads setup)
7. Go to the Product Feed page with the opening submission success modal: `/wp-admin/admin.php?page=wc-admin&guide=submission-success&path=%2Fgoogle%2Fproduct-feed`.
   - Without setting up ads, the content on the modal should have two pages.
   - Locally change the [`adsSetupComplete` value](https://github.com/woocommerce/google-listings-and-ads/blob/3314afc4782763d782d4d212c59d01237a64bce9/src/Admin/Admin.php#L128) to `true` and refresh page to simulate the finished ads setup. Check if the modal has the same copywriting as it is on Figma.

### Changelog entry

> Add - The disclaimer of Comparison Shopping Service of the accounts setup of onboarding flow.
> Add - The submission success modal on the Product Feed page after the onboarding is completed along with paid ads setup.
> Update - The FAQs in the paid ads setup and the campaign setup page.
> Tweak - Use different titles for the free listings setup of the onboarding and editing pages.
